src/jgikbase/test/idmapping/storage/mongo/test_id_mapping_mongo_storage.py

```diff
@@ -545,6 +565,20 @@ def fail_add_mapping(idstorage, pOID, sOID, expected):
         assert_exception_correct(got, expected)
 
 
+def test_remove_mapping_fail_input_None(idstorage):
```

Comment:
Do you have tests for when any of the object or namespace IDs don't exist in the database?

Reply:
No, it's not really a problem. The user didn't want that mapping to exist, and it doesn't.